app/src/main/java/org/wikipedia/page/edit_history/EditHistoryListActivity.kt

```diff
@@ -0,0 +1,139 @@
+package org.wikipedia.page.edit_history
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.View.OnClickListener
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.activity.viewModels
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView.Adapter
+import androidx.recyclerview.widget.RecyclerView.ViewHolder
+import org.wikipedia.R
+import org.wikipedia.activity.BaseActivity
+import org.wikipedia.commons.FilePageActivity
+import org.wikipedia.databinding.ActivityEditHistoryBinding
+import org.wikipedia.dataclient.mwapi.MwQueryPage.Revision
+import org.wikipedia.diff.ArticleEditDetailsActivity
+import org.wikipedia.page.EditHistoryListViewModel
+import org.wikipedia.page.PageTitle
+import org.wikipedia.util.DateUtil
+import org.wikipedia.util.Resource.Success
+import org.wikipedia.views.DefaultViewHolder
+
+class EditHistoryListActivity : BaseActivity() {
+
+    private lateinit var binding: ActivityEditHistoryBinding
+    private lateinit var editHistoryListAdapter: EditHistoryListAdapter
+    private lateinit var pageTitle: PageTitle
+    private val viewModel: EditHistoryListViewModel by viewModels()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityEditHistoryBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+        pageTitle = intent.getParcelableExtra(INTENT_EXTRA_PAGE_TITLE)!!
+        viewModel.fetchData(pageTitle)
```

Comment:
One of the benefits of using a ViewModel is that network calls don't need to be repeated unnecessarily, e.g. when the activity is recreated due to screen rotation, etc.
The way you have it here, the data will be fetched every time the activity is recreated, even if the ViewModel already contains valid data for this `pageTitle`.